python/servo/devenv_commands.py

```diff
@@ -87,11 +87,9 @@ def update_cargo(self, params=None, package=None, all_packages=None):
             print("flag or update all packages with --all-packages (-a) flag")
             sys.exit(1)
 
-        for cargo_path in CARGO_PATHS:
-            with cd(cargo_path):
-                print(cargo_path)
-                call(["cargo", "update"] + params,
-                     env=self.build_env())
+        with cd(self.context.topdir):
+            call(["cargo", "update"] + params,
+                 env=self.build_env())
```

Comment:
GH won't let me leave a comment there, but does this need to be updated in `./mach fetch` as well? Can `CARGO_PATHS` be deleted?